screens/EventDetails/EventDetails.tsx

```diff
@@ -0,0 +1,258 @@
+import { CurrentUserEvent, isAttendingEvent } from "@lib/events"
+import React, { useState } from "react"
+import { Ionicons } from "@expo/vector-icons"
+import { Image, ScrollView, StyleSheet, TouchableOpacity, View } from "react-native"
+import { BodyText, Caption, Headline, Title } from "@components/Text"
+import ExpandableText from "@screens/EventDetails/ExpandableText"
+import { Divider } from "react-native-elements"
+import EventMapSnippet from "./EventMapSnippet"
+import TimeSection from "./TimeSection"
+import LocationSection from "./LocationSection"
+import AttendeeSection from "./AttendeeSection"
+import ChatSection from "./ChatSection"
+import { OutlinedButton, PrimaryButton } from "@components/common/Buttons"
+import { CalendarEvent } from "@lib/Calendar"
+import { EventMapDetails, openMap} from "@lib/ExternalMap"
+import Toast from "react-native-root-toast"
+import { ButtonStyles } from "@lib/ButtonStyle"
+
+export type EventDetailsProps = {
+  event: CurrentUserEvent
+}
+
+const MARKER_SIZE = 44
+const BOTTOM_TAB_HEIGHT = 80
+
+const EventDetails = ({ event }: EventDetailsProps) => {
+  const isFriend = false
+  const [requestSent, setRequestSent] = useState(isFriend)
+  const calenderEvent: CalendarEvent = {
+    duration: event.dateRange,
+    id: event.id,
+    description: event.description!,
+    coordinates: event.coordinates,
+    title: event.title,
+    bottomTabHeight: BOTTOM_TAB_HEIGHT
+  }
+  const mapDetails: EventMapDetails = {
+    coordinates: event.coordinates,
+    placemark: event.placemark
+  }
+
+  const sendFriendRequest = () => {
+    setRequestSent(true)
+  }
+
+  const openMapWithoutDirections = () => {
+    openMap(mapDetails)
+  }
+
+  return (
+    <View>
+      <ScrollView
+        style={[styles.container, styles.spacing]}
+        contentContainerStyle={{ flexGrow: 1 }}
+        nestedScrollEnabled={true}
+      >
+        <Title style={styles.textColor}>{event.title}</Title>
+        <View style={[styles.flexRow, { marginVertical: 16 }]}>
+          <Image
+            style={[styles.image]}
+            source={require("../../assets/icon.png")}
+          />
+          <View style={{ flex: 1 }}>
+            <View style={styles.flexRow}>
+              <Headline style={styles.textColor}>{event.host.username}</Headline>
+              {!isFriend && 
```

Comment:
We'll reuse this profile image and name layout a lot in the app, so it would be good to have in a separate and configurable component